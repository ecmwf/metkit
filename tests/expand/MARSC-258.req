retrieve,
    class=rd,
    stream=lwda,
    levtype=pl,
    type=fc,
    expver=h3f7,
    step=all,
    anoffset=9,
    time=00,
    date=20181101,
    target=data.reference 