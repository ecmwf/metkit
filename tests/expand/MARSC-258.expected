RETRIEVE,
    CLASS      = RD,
    TYPE       = FC,
    STREAM     = LWDA,
    EXPVER     = h3f7,
    REPRES     = SH,
    LEVTYPE    = PL,
    LEVELIST   = 1000/850/700/500/400/300,
    PARAM      = 129,
    DATE       = 20181101,
    TIME       = 0000,
    STEP       = ALL,
    ANOFFSET   = 9,
    DOMAIN     = G,
    TARGET     = data.reference 